tests/test_TFUtil.py

```diff
@@ -2783,6 +2783,34 @@ def body(t, _x):
   print("magic (totally arbitrary) res:", session.run(x))
 
 
+def test_flatten_with_seq_len_mask():
+  import numpy as np
+  import numpy.testing as npt
+  batch_dim_axis = 0
+  time_dim_axis = 2
+  x = tf.constant(np.random.rand(10, 16, 11, 16))
+  seq_lens = tf.constant([11, 11, 11, 11, 11, 11, 11, 11, 11, 11])
```

Comment:
That does not make any sense. The whole test is senseless with such seq lens here.